packages/ui/src/views/labels/components/labels-list-view.tsx

```diff
@@ -55,31 +58,33 @@ export const LabelsListView: FC<LabelsListViewProps> = ({
         iconName="no-data-branches"
         title={t('views:noData.labels', 'No labels yet')}
         description={[t('views:noData.createLabel', 'Create a new label to get started.')]}
-        primaryButton={{
-          label: t('views:projectSettings.newLabels', 'Create label'),
-          to: 'create'
-        }}
+        primaryButton={{ label: t('views:projectSettings.newLabels', 'Create label'), to: 'create' }}
       />
     )
   }
 
+  const isSmallWidth = widthType === 'small'
+
   return (
     <Table.Root tableClassName="table-fixed" variant="asStackedList">
       <Table.Header>
         <Table.Row>
-          <Table.Head>
+          <Table.Head className={cn('w-1/4', { 'w-4/12': isSmallWidth })}>
             <span className="pl-[22px]">{t('views:labelData.table.name', 'Name')}</span>
           </Table.Head>
-          <Table.Head>{t('views:labelData.table.created', 'Created in')}</Table.Head>
-          <Table.Head>{t('views:labelData.table.description', 'Description')}</Table.Head>
-          <Table.Head />
+          <Table.Head className="w-1/4">{t('views:labelData.table.created', 'Created in')}</Table.Head>
+          <Table.Head className={cn('w-1/2', { 'w-5/12': isSmallWidth })}>
+            {t('views:labelData.table.description', 'Description')}
+          </Table.Head>
+          <Table.Head className="w-[54px]"></Table.Head>
```

Comment:
Does a self-closing tag work here?
```suggestion
          <Table.Head className="w-[54px]" />
```